config/routes.rb

```diff
@@ -672,7 +672,7 @@ def call(env)
         end
 
         scope module: :services do # this api has a knack for inconsistency
-          resources :backend_apis, controller: :backend_api_configs, except: %i[new edit], defaults: { format: :json }
+          resources :backend_api_configs, path: :backend_usages, as: :backend_usages, controller: :backend_usages, except: %i[new edit], defaults: { format: :json }
```

Comment:
Can't we change it to, 
```suggestion
          resources :backend_usages, except: %i[new edit], defaults: { format: :json }
```

that means having to `grep . backend_api_config -Rl | xargs sed -i -e 's|backend_api_config|backend_usage|g' `